osgtest/tests/test_24_tomcat.py

```diff
@@ -43,12 +34,39 @@ def test_04_config_tomcat_endorsed_jars(self):
             new_contents = old_contents + "\n" + line
             files.write(tomcat.conffile(), new_contents, owner='tomcat')
 
+    def test_04_configure_gratia(self):
+        core.skip_ok_unless_installed(tomcat.pkgname(), 'gratia-service')
+        command = ('/usr/share/gratia/configure_tomcat',)
+        core.check_system(command, 'Unable to configure Gratia.')
+
     def test_05_start_tomcat(self):
```

Comment:
Yes: including it in `test_23_gratia.py` results in config changes being overwritten by the earlier tests in `test_24_tomcat.py`.